### ISSUE TITLE
Use *-dev packages instead of building from source?

### ISSUE THREAD

**Issue by summersab** [ONLYOFFICE/build_tools#401]
**Describe your problem**:
The `build_tools` package builds a LOT of the supplementary packages (QT, OpenSSL, V8, etc) from source. The README mentions that the tools were built for Ubuntu 14.04, and I assume many dev packages weren't in the repositories at the time. Would there be an issue with modifying the scripts to support using existing dev packages instead of building them all from source? For example:
`libboost-dev`
`libcrypto++-dev`
`libglew-dev`
`libgumbo-dev`
`libhunspell-dev`
`libicu-dev`
`libssl-dev`
`libv8-dev`
`ninja-build`

**Comment by ShockwaveNN** [comment]
Hi, your question is legit and I like it, but currently, we cannot change dev dependencies to packages for several reasons

1. We need to patch some of our dependencies to work well with our codebase - and those patches potentially may break on even minor updates of dev dependency, so we stick to a very specific version of source packages
2. We need to support a lot of OSes and even minor updates of some dependency can break something on some old OS, we already got a lot of problems supporting v8 on old Ubuntu 14.04 systems
3. We got some promises for some of our big clients to support some specific OSes, based on very old debian-ubuntu, even older than v14.04

Combining those factors and some other factors I cannot disclouse - we decided to use compiling packages from source
